<launch>
	<!-- JOY NODE -->
	<node respawn="true" pkg="joy"
		type="joy_node" name="joy_node" >
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
	</node>
	<!-- PS3 JOYSTICK PARAMETERS -->
	<param name="axis_linear_L" value="1" type="int"/>
	<param name="axis_linear_R" value="3" type="int"/>
	<!-- SCALAR PARAMETERS -->
	<param name="scale_linear_L" value="9" type="double"/>
	<param name="scale_linear_R" value="9" type="double"/>
	<!-- TF BROADCASTER NODE -->
	<node pkg="agv" type="agvTeleop"  name="agv_teleop" ></node>
</launch>

